### DISCUSSION TITLE
Implementation of Syn flood attack in 5G network in Oment++

### DISCUSSION THREAD

**Discussion by kermanizahra** [omnetpp/omnetpp#1263] [Q&A]
Hello, I don't know what modules to use and how to use it and the general idea I have about it has encountered a problem. Can anyone guide me about this project?

**Comment by Muhammaduazir69** [comment]
Hello @kermanizahra Yes i can guide you about this project, i can help you setup modules configurations in omnet++ with simu5g & attack scenario.

**Comment by kermanizahra** [comment]
Thank you so much...
My project is about the implementation of syn flood attack on 5g network in
omnet++. I have written an attack module in .cc and .h file, but when the
project is run, it has this error. I checked its names and spelling several
times and made and clean were done, but the error was not fixed.

در تاریخ جمعه ۱۵ نوامبر ۲۰۲۴،‏ ۱۳:۰۵ Muhammaduazir69 <
***@***.***> نوشت:

**Comment by kermanizahra** [comment]
در تاریخ شنبه ۱۶ نوامبر ۲۰۲۴،‏ ۱۸:۳۹ Zahra Kermani <
***@***.***> نوشت:

**Comment by Muhammaduazir69** [comment]
Hello @kermanizahra let's connect to see what errors you're facing while implementing syn flood attack module, check my profile to contact me.

**Comment by kermanizahra** [comment]
Hello again, thank you very much
How can I contact you??

در تاریخ یکشنبه ۱۷ نوامبر ۲۰۲۴،‏ ۰۴:۱۲ Muhammad Uzair <
***@***.***> نوشت:

**Reply by Muhammaduazir69** [reply]
@kermanizahra check my github profile & you will find email in bio!
